packages/sanity/src/_internal/cli/commands/documents/validateDocumentsCommand.ts

```diff
@@ -19,10 +19,10 @@ Options
   --dataset <name> Override the dataset used. By default, this is derived from the given workspace
   --format <pretty|ndjson|json> The output format used to print the found validation markers and report progress
   --level <error|warning|info> The minimum level reported out. Defaults to warning.
-  --max-custom-validation-concurrent <number> Specify how many custom validators can run concurrently. Defaults to 5.
+  --max-custom-validation-concurrency <number> Specify how many custom validators can run concurrently. Defaults to 5.
```

Comment:
Minor, but for consistency, we should add a period to the end of the descriptions for the other options (`y`, `workspace`, etc).